### ISSUE TITLE
Include an image of Sawppy the Rover

### ISSUE THREAD

**Issue by sethfischer** [sethfischer/osr#1]
Include an image of _Sawppy the Rover_ in the _Related projects_ section.

# Suggested image

https://cdn.hackaday.io/images/8367701559592489700.jpg

![8367701559592489700](https://user-images.githubusercontent.com/2492233/156867303-1c2d373d-d9d7-495c-aa51-c9f93bf90ef7.jpg)

**Comment by sethfischer** [comment]
@Roger-random May I have your permission to include this photo of yours in the _Related projects_ section?

I would provide links to your project add the following EXIF metadata tags (or similar) to the JPEG image:

```
Artist           : Roger Cheng
Attribution Name : Roger Cheng
Attribution URL  : https://github.com/Roger-random/Sawppy_Rover
License          : https://spdx.org/licenses/MIT.html
Description      : Sawppy is a motorized model of Mars rovers Curiosity and Perseverance.
Rights           : MIT
Usage Terms      : Copyright Roger Cheng
Comment          : Sawppy the Rover
```

**Comment by Roger-random** [comment]
The EXIF additions look great. Permission granted, and thanks for asking.

**Comment by sethfischer** [comment]
Thank you @Roger-random

**Comment by sethfischer** [comment]
Closed by c6792072d329fa0d8502397a5a812eebdba755cc
